### PR TITLE
Lvjavs 671 admin cannot deactivate room

### PR DESCRIPTION
**Pull request by andryspo**
add cascade bookings cancelling

### CODE REVIEW

File: src/main/java/ua/softserveinc/tc/controller/admin/EditRoomController.java [hunk 1]
```diff
@@ -3,10 +3,7 @@
 import com.google.gson.Gson;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.*;
```
Comment (thrytsko-amplify) [inline src/main/java/ua/softserveinc/tc/controller/admin/EditRoomController.java]: It's a bad practice to use * in imports

File: src/main/java/ua/softserveinc/tc/dao/impl/BookingDaoImpl.java [hunk 2]
```diff
@@ -5,18 +5,17 @@
...
 
 import javax.persistence.EntityManager;
 import javax.persistence.PersistenceContext;
-import javax.persistence.criteria.CriteriaBuilder;
-import javax.persistence.criteria.CriteriaQuery;
-import javax.persistence.criteria.Predicate;
-import javax.persistence.criteria.Root;
-import java.util.Collections;
-import java.util.List;
+import javax.persistence.criteria.*;
```
Comment (thrytsko-amplify) [inline src/main/java/ua/softserveinc/tc/dao/impl/BookingDaoImpl.java]: same here

Comment (andryspo) [inline src/main/java/ua/softserveinc/tc/dao/impl/BookingDaoImpl.java]: ok  I will refactor it

File: src/main/java/ua/softserveinc/tc/controller/admin/EditRoomController.java (hunk 1 above)
Comment (andryspo) [inline src/main/java/ua/softserveinc/tc/controller/admin/EditRoomController.java]: ok I will refactor it

**Pull request merged**